README.md

```diff
@@ -18,14 +18,21 @@ Conda instructions are provided in more detail, but you may also use `pip` and `
 
 ### Install with pip
 
-To use `pip` to install TorchServe and the model archiver:
+1. Install Java 11
 
-``` bash
-pip install torch torchtext torchvision sentencepiece psutil future
-pip install torchserve torch-model-archiver
-```
+    ```bash
+    sudo apt-get install openjdk-11-jdk
+    ```
+
+1. Use `pip` to install TorchServe and the model archiver:
+
+    ``` bash
+    pip install torch torchtext torchvision sentencepiece psutil future
+    pip install torchserve torch-model-archiver
+    ```
 
 ### Install with Conda
+**Note:** For Conda, Python 3.8 is required to run Torchserve
```

Comment:
I don't think this is true. I tested it on 3.6, 3.7, 2.7... vanilla installs and upgrading the existing conda envs on DLAMI. Sounds like testing is needed again, but anyways... don't think this is accurate and shouldn't be the case anyway...

Reply:
Even if we have been testing for many versions and different combinations, it is in our favor to explicitly call out one version number for each package that we are confident having tested with, and against which we are testing while running our UTs, CI/CD, Integration tests etc.

Reply:
Such a scoping is useful for our users so that they are using a consistent environment, which has already been tested.